app/collections/tracks.js

```diff
@@ -1,24 +1,49 @@
 import Backbone from 'backbone';
 import Track from '../models/track';
+import cozysdk from 'cozysdk-client';
 
 
 const Tracks = Backbone.Collection.extend({
     model: Track,
+
+    initialize: function(models, options) {
+        this.on('add', this.onAdd, this);
+        if (options) {
+            this.type = options.type;
```

Comment:
You can better pass `options = {}` as defaults param if your function definition and just `this.type = options.type`. It'll be the passed value in the `options` hash or `undefined` if the `type` key isn't available.